tests/test_docs_generation_output.py

```diff
@@ -128,6 +131,25 @@ def test_files_generated_for_each_optional_project_definition_property(
     assert len(errors) == 0, "\n".join(errors)
 
 
+def test_files_generated_for_each_entity_definition(runner, temp_dir):
+    runner.invoke(["--docs"])
+    project_definition_path = Path(temp_dir) / "gen_docs" / "project_definition_V20"
+    errors = []
+
+    model_json = model_json_schema(DefinitionV20, schema_generator=GenerateJsonSchema)
+    mapping = model_json["properties"]["entities"]["additionalProperties"][
+        "discriminator"
+    ]["mapping"]
+    for property_name, _ in mapping.items():
+        if not (
+            project_definition_path
+            / f"definition_entities_{property_name.replace(' ', '_')}.txt"
+        ).exists():
+            errors.append(f"Section `{property_name}` was not properly generated")
+
+    assert len(errors) == 0, "\n".join(errors)
```

Comment:
Could you add more assertions/test cases to check if proper number of files is generated and their content is ok?